airbyte-integrations/connectors/source-woocommerce/integration_tests/configured_catalog.json

```diff
@@ -385,7 +391,7 @@
                     "type": ["null", "integer"]
                   },
                   "quantity": {
-                    "type": ["null", "integer"]
+                    "type": ["null", "string"]
```

Comment:
Why did we change it to string?

Reply:
Quantity is coming in float. so, it is failing the normalization